recipes/sphericart/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "sphericart" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/sphericart-{{ version }}.tar.gz
+    sha256: 2f3d2672a20efd7cf400cce42d0648f04d16fe8abaf834d167a1b45ef2f87e46
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "11.8"]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("cuda") }}
+    - {{ stdlib("c") }}
+    - cmake
+    - cuda-nvcc
+    - cuda-toolkit
+    - make
+    - ninja
+    - pip
+    - pybind11
+    - python
+    - pytorch
+    - setuptools
```

These should not be here or guarded with a `build_platform != target_platform` guard.
```suggestion
    - pip
    - pybind11
    - python
    - pytorch
    - setuptools
```